homeassistant/components/connectedcars/const.py

```diff
@@ -0,0 +1,36 @@
+"""Constants for the ConnectedCars.io integration."""
+
+ATTRIBUTION = "Data provided by ConnectedCars.io"
+
+ATTR_API_USER_EMAIL = "email"
+ATTR_API_USER_FIRSTNAME = "first_name"
+ATTR_API_USER_LASTNAME = "last_name"
+
+ATTR_API_VEHICLE_FUELLEVEL = "vehicle_fuel_level"
+ATTR_API_VEHICLE_FUELPERCENTAGE = "vehicle_fuel_percentage"
+ATTR_API_VEHICLE_ID = "vehicle_id"
+ATTR_API_VEHICLE_LICENSEPLATE = "vehicle_license_plate"
+ATTR_API_VEHICLE_MAKE = "vehicle_make"
+ATTR_API_VEHICLE_MODEL = "vehicle_model"
+ATTR_API_VEHICLE_NAME = "vehicle_name"
+ATTR_API_VEHICLE_ODOMETER = "vehicle_odometer"
+ATTR_API_VEHICLE_POS_LATITUDE = "vehicle_position_latitude"
+ATTR_API_VEHICLE_POS_LONGITUDE = "vehicle_position_longitude"
+ATTR_API_VEHICLE_VIN = "vehicle_vin"
+ATTR_API_VEHICLE_VOLTAGE = "vehicle_voltage"
+
+ATTR_ICON = "icon"
```

Comment:
Some of those are already available via `homeassistant.const` please re-use those as much as possible.